src/types.ts

```diff
@@ -22,3 +25,14 @@ export const signedApiResponseSchema = z.object({
 });
 
 export type LocalSignedData = Pick<SignedData, 'encodedValue' | 'signature' | 'timestamp'>;
+
+export interface DataFeedSingle {
```

Comment:
I think it would be better to call this `Beacon` and have the `DecodedDataFeed` specified as:
```ts
export interface DecodedDataFeed {
  dataFeedId: string;
  beacons: Beacon[];
}
```